### PR TITLE
Scenario 6a

### PR DESCRIPTION
**Pull request by nkrinner**
Needs manual SBD setup as described in https://etherpad.nue.suse.com/p/cloud-setup-qa-scenarios
and also manual setup of an nfs server for shared storage. The servers ip address needs to be added in yaml

### CODE REVIEW

**Comment by jsuchome** [comment]
Why is there 2c setup? That has been already merged...

**Comment by nkrinner** [comment]
Oh, sorry  it seems I messed up my branch while rebasing...

File: scripts/scenarios/cloud6/qa-scenario-6a-sbd-kvm-docker.yaml [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+        shared:
+          device: <nfs-server-ip>:/exportpg
+          fstype: nfs
+  deployment:
+    elements:
+      database-server:
+      - cluster:services
+- barclamp: rabbitmq
+  attributes:
+    password: uz8FaasWz9st
```
Comment (jsuchome) [inline scripts/scenarios/cloud6/qa-scenario-6a-sbd-kvm-docker.yaml]: This should be removed

File: scripts/scenarios/cloud6/qa-scenario-6a-sbd-kvm-docker.yaml [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+      pacemaker-cluster-member:
+      - "@@controller1@@"
+      - "@@controller2@@"
+      - "@@controller3@@"
+      hawk-server:
+      - "@@controller1@@"
+      - "@@controller2@@"
+      - "@@controller3@@"
+# WARNING: no proposals exist for nfs_client barclamp
+# WARNING: no proposals exist for suse_manager_client barclamp
```
Comment (jsuchome) [inline scripts/scenarios/cloud6/qa-scenario-6a-sbd-kvm-docker.yaml]: I think these comments are not needed

File: scripts/scenarios/cloud6/qa-scenario-6a-sbd-kvm-docker.yaml [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+    elements:
+      database-server:
+      - cluster:services
+- barclamp: rabbitmq
+  attributes:
+    password: uz8FaasWz9st
+    ha:
+      storage:
+        shared:
+          device: <nfs-server-ip>:/exportrabbit
```
Comment (jsuchome) [inline scripts/scenarios/cloud6/qa-scenario-6a-sbd-kvm-docker.yaml]: I'm not sure about this one. But maybe it is better than using some specific address that is QA specific.

File: scripts/scenarios/cloud6/qa-scenario-6a-sbd-kvm-docker.yaml [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+          fstype: nfs
+  deployment:
+    elements:
+      rabbitmq-server:
+      - cluster:services
+- barclamp: keystone
+  attributes:
+    database_instance: default
+    rabbitmq_instance: default
```
Comment (jsuchome) [inline scripts/scenarios/cloud6/qa-scenario-6a-sbd-kvm-docker.yaml]: Those lines with 'default' value could be removed (in the whole file)

File: scripts/scenarios/cloud6/qa-scenario-6a-sbd-kvm-docker.yaml [hunk 5]
```diff
@@ -0,0 +1,231 @@
...
+  deployment:
+    elements:
+      heat-server:
+      - cluster:services
+- barclamp: ceilometer
+  attributes:
+    rabbitmq_instance: default
+    database_instance: default
+    keystone_instance: default
+    service_password: F4nGhFIE7D7V
```
Comment (jsuchome) [inline scripts/scenarios/cloud6/qa-scenario-6a-sbd-kvm-docker.yaml]: Remove the password

**Comment by nkrinner** [comment]
Comments addressed

File: scripts/scenarios/cloud6/qa-scenario-6a-sbd-kvm-docker.yaml [hunk 6]
```diff
@@ -0,0 +1,194 @@
+---
+proposals:
+- barclamp: pacemaker
+  name: services
+  attributes:
+    stonith:
+      mode: sbd
+      sbd:
+        nodes:
+          d52-54-77-77-77-01.v6.cloud.suse.de:
```
Comment (jsuchome) [inline scripts/scenarios/cloud6/qa-scenario-6a-sbd-kvm-docker.yaml]: Please use aliases here

File: scripts/scenarios/cloud6/qa-scenario-6a-sbd-kvm-docker.yaml [hunk 7]
```diff
@@ -0,0 +1,194 @@
+---
+proposals:
+- barclamp: pacemaker
+  name: services
+  attributes:
+    stonith:
+      mode: sbd
```
Comment (jsuchome) [inline scripts/scenarios/cloud6/qa-scenario-6a-sbd-kvm-docker.yaml]: Actually... description says it should use IPMI, so there should not be a need for sbd (not even the nodes below, I guess)

**Comment by nkrinner** [comment]
Wow, I was really sure that SBD was required here, need to change that...

**Comment by nkrinner** [comment]
More comments addressed. hopefully this is right now

File: scripts/scenarios/cloud6/qa-scenario-6a-ipmi-kvm-docker.yaml [hunk 8]
```diff
@@ -0,0 +1,195 @@
+---
+proposals:
+- barclamp: pacemaker
+  name: services
+  attributes:
+    stonith:
+      mode: ipmi_barclamp
+      sbd:
```
Comment (jsuchome) [inline scripts/scenarios/cloud6/qa-scenario-6a-ipmi-kvm-docker.yaml]: Is this sbd subsection needed?

**Comment by jsuchome** [comment]
Bascially +1. Those sbd parts do not hurt (although I think it would be better without them)

**Comment by vuntz** [comment]
+1

**Comment by aplanas** [comment]
+1

**Pull request merged**